### PR TITLE
feat(helm): add loki-stack to KAI chart

### PR DESCRIPTION
**Pull request by IGZangelcostales**
## Description
Add Loki and Grafana for KAI management logs

## Motivation and Context
Initially, we needed a Loki to have a log aggregation system for KAI, then we added Grafana since it makes sense to have a place where we can have that logs observability.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other changes (ci configuration, documentation or any other kind of changes)

## Checklist:

- [ ] I have created tests for my code changes, and the tests are passing.
- [x] I have executed the pre-commit hooks locally.
- [ ] My change requires a change to the documentation (create a new issue if the documentation has not been updated).
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: scripts/kaictl/cmd_etchost.sh [hunk 1]
```diff
@@ -47,6 +48,7 @@ show_etc_hosts() {
   echo_info "👇 Add the following lines to your /etc/hosts"
   echo
   echo "$INGRESS_IP api.kai.local"
+  echo "$INGRESS_IP grafana.kai.local"
```
Comment (alpiquero) [inline scripts/kaictl/cmd_etchost.sh:51]: Can we change this to something less stack specific like metrics?
Maybe @dfinteligenz has a suggestion for the name?

File: helm/kai/values.yaml [hunk 2]
```diff
@@ -796,3 +796,84 @@ minio:
...
+    # -- Deployment Strategy
+    deploymentStrategy:
+      type: Recreate
+    image:
+    # -- Grafana version
+      tag: 10.2.0
+    # -- Admin user name
+    adminUser: admin
+    # -- Set admin password (ommited if existingSecret is set)
+    adminPassword: "123456"
```
Comment (alpiquero) [inline helm/kai/values.yaml:843]: Replace this for an empty value. The Grafana subchart uses a random value for this when it's empty

**Review by alpiquero — DISMISSED** [review]

File: scripts/kaictl/cmd_etchost.sh (hunk 1 above)
Comment (IGZangelcostales) [reply scripts/kaictl/cmd_etchost.sh]: done

File: helm/kai/values.yaml (hunk 2 above)
Comment (IGZangelcostales) [reply helm/kai/values.yaml]: done

**Pull request merged**